conman/routes/migrations/0001_initial.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import polymorphic_tree.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contenttypes', '0002_remove_content_type_name'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Route',
+            fields=[
+                ('id', models.AutoField(auto_created=True, verbose_name='ID', primary_key=True, serialize=False)),
+                ('slug', models.SlugField(help_text='\n        Used to create the location of the Route. The Root Route needs\n        "slug" to be blank; all other Routes need a value unique to the parent.\n        It can only contain letters, numbers, underscores, or hyphens.\n    ', default='', max_length=255)),
```

Comment:
I'm not sure the [`slug.help_text`](https://github.com/meshy/django-conman/blob/master/conman/routes/models.py#L55-L59) is correctly defined. Perhaps it should be using [`textwrap.dedent`](https://docs.python.org/3.4/library/textwrap.html#textwrap.dedent)?